sbws/lib/stem_bandwidth_file.py

```diff
@@ -0,0 +1,67 @@
+"""Stem's Bandwidth file parser part.
```

Comment:
Hmm, is this the right way to do this? Just include a file from stem? Doesn't this mean we need to keep track of stem's revisions to this file? :/

Also if this is the right way to do this, then perhaps we should mention which revision of this file this is (commit id). Also, why does it say "To be removed on Stem's release 1.8.0"?

Reply:
> Hmm, is this the right way to do this? Just include a file from stem? Doesn't this mean we need to keep track of stem's revisions to this file? :/

Only until the next release, i think there're stem's releases every ~6 months.
And so far this file has only changed when after adding more headers to `sbws` i create an issue in `stem` so it gets added too.
With this change, instead i could directly create a PR in stem and duplicate it in `sbws` until the next release.

Not sure is there's a better way, but i definitely don't want to wait 8 months again to do a sbws release waiting for a stem release and it's not a solution to change the dependency to stem's git master because that does not work for system packages (Debian).

> 
> Also if this is the right way to do this, then perhaps we should mention which revision of this file this is (commit id).

Do you think it would be better i just add the stem's bandwidth_file.py as it is currently?.
I can add the commit id (in v3bwfile if i add the stem's bandwidth_file.py as it's)

> Also, why does it say "To be removed on Stem's release 1.8.0"?

Because that's the next Stem's release, i can add this comment somewhere. Where it would fit better?.

Reply:
if i add the file as it is, i also need to copy the `__init__.py`
Thinking more about this, i think it's better to leave only that part, to follow better what i substitute.
I could also do a `try import` and if there's no `bandwidth_file.py` in stem then leave the old code + the refactor.
But i think duplicating all that code when as i said, the file won't change unless someone open a ticket, and there'll be a new stem release soon.

Reply:
See small changes in fixup